### ISSUE TITLE
Modernise SUISA Sendemeldung infra

### ISSUE THREAD

**Issue by hairmare** [radiorabe/suisa_sendemeldung#69]
I've recently shown nowplaying quite some 🐍 love and want to ensure that SUISA sendemeldung also benefits from that.

* [x] configure CI (actions, dependabot) https://github.com/radiorabe/suisa_sendemeldung/pull/68
* [x] configure CD (pypi, semantic-release) #169
* [x] implement testing #170
* [x] deprecate python 3.8, switch to 3.11 on ubi9 #306
* [x] switch to poetry instead of soon to be deprecated setuptools #306
* [x] replace twine with poetry for pypi publishing  #306
* [x] introduce quality gates (mypy, ...) https://github.com/radiorabe/suisa_sendemeldung/pull/395
* [x] ~~harmonize logging with otel (+ further telemetry like traces and metrics)~~
* [x] use [radiorabe/python-minimal](https://github.com/radiorabe/container-image-python-minimal) container image #118 
* [x] cleanup README.md 
  * should also dox #289 
* [ ] #644

Basically, what i wat for suisa_sendemeldung is for it to be in a state where consistent and regular rollouts + upgrades are a thing (like with nowplaying which is currently in a quite ok state).
